1.Core/Helpers/Logger.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Serilog;
+
+namespace Todoly.Core.Helpers;
+public static class Logger
+{
+    private static ILogger? _logger;
+
+    public static ILogger Instance
+    {
+        get
+        {
+            if (_logger == null)
+            {
+                _logger = new LoggerConfiguration()
+                            .MinimumLevel.Debug()
+                            .WriteTo.File("logs.txt", outputTemplate: "{Timestamp:yyy-MM-dd HH:mm:ss.fff} | {Level:u3} | {Message} {NewLine}", rollingInterval: RollingInterval.Day)
+                            .CreateLogger();
+            }
+
+            return _logger;
+        }
+    }
+    public static void ConfigureLogger()
+    {
+        Log.Logger = new LoggerConfiguration()
+            .MinimumLevel.Debug()
+            .Enrich.FromLogContext()
+            .WriteTo.File("logs\\log.txt", rollingInterval: RollingInterval.Day, retainedFileCountLimit: 31, fileSizeLimitBytes: 10485760, buffered: true)
```

Comment:
Actually we might need to store the logs in different files when running the tests locally, one for each execution. So instead of having a rolling interval for each day, we could have a bunch of files identified by a timestamp maybe.

Reply:
Alright, so it would be a file for each feature right?

Reply:
Nope, for each test execution